### ISSUE TITLE
extend color.ini functionality to PIP windows

### ISSUE THREAD

**Issue by invisiblefmx** [spicetify/spicetify-cli#2836]
### 📝 Provide a description of the new feature

Spotify for Windows (64 Bit)
1.2.31.1205.g4d59ad7c
Spicetify v2.31.3

Since there is a new pip player activated in the new spotify release it would be great if themes had an easy way to change its colors to fit the theme. Dark grey with black songtitle and black buttons just looks bad and is not very readable.

![image](https://github.com/spicetify/spicetify-cli/assets/157469657/cdaa8bf5-c9b2-472a-a79f-efa8a51a89b8)

![image](https://github.com/spicetify/spicetify-cli/assets/157469657/1a0b2af4-7e2c-4f51-a203-df888fc2a360)

**Comment by ohitstom** [comment]
I agree, they must be using some different variable naming conventions or something - I don't see why we couldn't patch them like we do with the rest of the global color vars.

**Comment by rxri** [comment]
I cannot get it to enable so I cannot work on it atm

**Comment by rxri** [comment]
Dependent on encore variable colors, not suitable to implement in v2

**Comment by ohitstom** [comment]
> Dependent on encore variable colors, not suitable to implement in v2

it looks like the only issue is that there is no existence of colors.css on created instances (perhaps we cant just modify the `pip-mini-player.css`) , also the way to enable PIP is by running on the chrome runtime not alloy and by enabling the exp feature!

![image](https://github.com/spicetify/spicetify-cli/assets/22730962/9a0814bd-0dc6-4f57-a010-6e75d4710be5)

**Comment by ohitstom** [comment]
having a quick look it seems no .html file exists and its done internally somewhere, to save use from webpack headache and stuff maybe we can just append the :root colors directly at the top of "pip-mini-player.css" during the patching stage? how does that sound to you @rxri

**Comment by rxri** [comment]
> having a quick look it seems no .html file exists and its done internally somewhere, to save use from webpack headache and stuff maybe we can just append the :root colors directly at the top of "pip-mini-player.css" during the patching stage? how does that sound to you @rxri

It's not worth it. v3 is on its way and will be released in ~2 months. This PiP is not released to everyone so I'm not going to merge it just yet, and it also requires chrome runtime which is rolling out I guess?

**Comment by ohitstom** [comment]
> > having a quick look it seems no .html file exists and its done internally somewhere, to save use from webpack headache and stuff maybe we can just append the :root colors directly at the top of "pip-mini-player.css" during the patching stage? how does that sound to you @rxri
> 
> It's not worth it. v3 is on its way and will be released in ~2 months. This PiP is not released to everyone so I'm not going to merge it just yet, and it also requires chrome runtime which is rolling out I guess?

may i ask what v3 entails? im not seeing branch, issue, or pr - just interested

**Comment by rxri** [comment]
<https://github.com/delusoire/bespoke>

**Comment by ohitstom** [comment]
> https://github.com/delusoire/bespoke

not sure how to feel about this, gonna turn everything upside down lol! good luck though
